CHANGELOG.md

```diff
@@ -1,11 +1,45 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+- Alignment position support (#44)
+
+## [2.2.0]
+
+### Added
+- Compact printing of alignments (#53)
+- Downstream tests (#70)
+- SSH secret to TagBot.yml (#71)
+- Zenodo badge (#c49bb0a3)
+- Doctests as part of the unit tests (#78)
+
+### Changed
+- Updated CI to use GitHub actions (#47, #77)
+- *master* set as the main development branch (#49, #50)
+- Updated chat links (#ebea200c)
+- Expanded test matrix, adding julia v1.6 (#77)
+- Incremented BioSequences compatibility to v3 (#72)
+- Incremented BioSymbols compatibility to v5 (#72)
+- Adjusted test matrix for minimum compatibility (#72)
+- Updated doctests (#72)
+
+### Removed
+- :exclamation: Reverted the use of *BioJulia* registry,
+  the package switched to [General Julia Registry](https://github.com/JuliaRegistries/General) (#48)
+- :exclamation: Removed Gitter chat links (#ebea200c)
+- :exclamation: Dropped support for julia less than v1.6 (#72)
+
+### Fixed
+- Doctests (#78)
+
+## 0.2.1 - 2022-10-22 [YANKED]
+
 ## [2.0.0]
```

Comment:
```suggestion

## [Unreleased]

## [2.2.0]

### Added
- Compact printing of alignments (#53)
- Downstream tests (#70)
- SSH secret to TagBot.yml (#71)
- Doctests as part of the unit tests (#78)

### Changed
- Updated chat links (#ebea200c)
- Expanded test matrix, adding julia v1.6 (#77)
- Incremented BioSequences compatibility to v3 (#72)
- Incremented BioSymbols compatibility to v5 (#72)
- Adjusted test matrix for minimum compatibility (#72)
- Updated doctests (#72)

### Removed
- :exclamation: Removed Gitter chat links (#ebea200c)
- :exclamation: Dropped support for julia less than v1.6 (#72)
- :exclamation: Removed alignment position support, as it was a breaking change (#44, #76)

### Fixed
- Doctests (#78)

## [2.1.0] - 2022-02-10 [YANKED]

### Added
- Zenodo badge (#c49bb0a3)

### Removed
- :exclamation: Dropped support for julia less than v1 (#50)

### Changed

## [2.0.1] - 2022-02-08 [UNREGISTERED]

### Added
- Alignment position support (#44)

### Changed
- Updated CI to use GitHub actions (#47)
- *master* set as the main development branch (#49, #50)

### Removed
- :exclamation: Reverted the use of *BioJulia* registry,
  the package switched to [General Julia Registry](https://github.com/JuliaRegistries/General) (#48)

## [2.0.0]
```

I think this is best way to reflect the state of the project.

Reply:
That's a better back-fill.